db/seeds.rb

```diff
@@ -18,6 +18,7 @@
   fap.phone_number = row["phone_number"]
   fap.website = row["website"]
   fap.website = "http://cnn.com" unless !fap.website.blank?
+  fap.user_id = 1;
```

Comment:
rather than hard coding a user_id here. i'd create a seed user above like `user = User.create(pass, in, attributes, here)` and then here you could do `fap.user = user`

Reply:
I can fix that too....makes sense.  I'll be working  some on this tomorrow - so I can get another commit up that addresses all these issues.